### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by dbanksdesign**
*Issue #, if available:*

*Description of changes:* Before making this repository public, we need a proper README that explains what this repository is and the current state of things. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +13,168 @@
 
 -->
 
----
+# Amplify UI 2.0
+
+Amplify UI 2.0 is an open-source UI library with cloud-connected components that are endlessly customizable, accessible, and can integrate into _any_ application. Amplify UI consists of:
+
+1. Connected components that simplify complex cloud-connected workflows, like Authenticator.
```
Comment (siegerts) [inline README.md:20]: Thoughts on this this being cloud platform specific (i.e. Amplify-connected)?      

Something like "Connected components that simplify complex Amplify workflows, like Authenticator."?

I'm thinking that if a customer lands on this repo and has never heard of Amplify then "cloud connected" could mean many different things, especially for those that are more frontend focused.

File: README.md [hunk 2]
```diff
@@ -15,10 +13,168 @@
...
+
+> Note: primitives not available for Angular yet
 
 ## Documentation
 
-The latest documentation is found here:
+- Docs for the latest stable version of Amplify UI can be found [here](https://docs.amplify.aws/ui).
+- Docs for the next version of Amplify UI can be found [here](https://ui.docs.amplify.aws).
+
+Improving our documentation and providing better and more interactive sample code is one of the focuses of the new Amplify UI. We also wanted to make it easier to keep our docs in sync with the UI code so now this repository will contain all the documentation, UI code, and end-to-end tests to ensure the correctness of the documentation.
```
Comment (siegerts) [inline README.md:100]: Maybe adding where in the repo these elements are located. For example "...all the documentation (/docs), UI code (/packages),...."

File: README.md [hunk 3]
```diff
@@ -15,10 +13,168 @@
...
+
+**Why is this a new repository?**
+
+We are planning to build more cloud-connected components across multiple frameworks and platforms and it makes more sense to have a unified UI repository with components for all frameworks and platforms.
+
+## Contributing
+
+Right now the best way to contribute is to try out the new Amplify UI and give us feedback. We want to make sure we are building the right tools for you to build amazing applications easily.
+
+## Status
```
Comment (siegerts) [inline README.md:135]: I think this section/table is too far down. As a developer, this is the first thing that I'd evaluate to determine if what I need is available or _in the works_.

File: README.md [hunk 4]
```diff
@@ -15,10 +13,168 @@
...
+
+As we continue to work on the new Amplify UI we will move UI-related issues in the amplify-js repository over here to work on them. We will continue to maintain major bug and security fixes for all existing UI packages and versions. New development for UI components will happen in this repository and eventually be published under the `@next` npm tag.
+
+**Why is this a new repository?**
+
+We are planning to build more cloud-connected components across multiple frameworks and platforms and it makes more sense to have a unified UI repository with components for all frameworks and platforms.
+
+## Contributing
+
+Right now the best way to contribute is to try out the new Amplify UI and give us feedback. We want to make sure we are building the right tools for you to build amazing applications easily.
```
Comment (siegerts) [inline README.md:133]: Where is the best place to provide feedback - issues/feat requests, discussions, discord?

File: README.md [hunk 5]
```diff
@@ -15,10 +13,168 @@
...
+## Looking for other products?
 
-> https://docs.amplify.aws/ui
+- [Amplify Console](https://docs.amplify.aws/console)
+- [Amplify JS](https://docs.amplify.aws/lib/q/platform/js)
+- [Amplify iOS](https://docs.amplify.aws/lib/q/platform/ios)
+- [Amplify Android](https://docs.amplify.aws/lib/q/platform/android)
+- [Amplify Flutter](https://docs.amplify.aws/lib/q/platform/flutter)
+- [Amplify CLI](https://docs.amplify.aws/cli)
+- [Amplify Community](https://amplify.aws/community/)
```
Comment (siegerts) [inline README.md:180]: cc @mauerbac thoughts on removing the community repo from this list? I'd opt not to include it right now.

File: README.md [hunk 6]
```diff
@@ -15,10 +13,168 @@
...
+```
+
+[Head over to the docs for more information on Angular usage](https://ui.docs.amplify.aws/ui/components/authenticator?platform=angular)
+
+> Note: primitives not available for Angular yet
 
 ## Documentation
 
-The latest documentation is found here:
+- Docs for the latest stable version of Amplify UI can be found [here](https://docs.amplify.aws/ui).
```
[the remark this answers is not in the document]
Comment (dbanksdesign) [reply README.md:97]: There will be, I am working on that plan right now.

**Pull request merged**